File: Bilibili/Extension/UI/UIImage/UIImageExtension.m
//
//  UIImageExtension.m
//  Extension
//
//  Created by LunarEclipse on 16/6/27.
//  Copyright © 2016年 LunarEclipse. All rights reserved.
//

#import "UIImageExtension.h"

@implementation UIImage (Extension)

- (UIImage *)centerStretchableImage
{
    return [self stretchableImageWithLeftCapWidth:self.size.width * 0.5 topCapHeight:self.size.height * 0.5];
}

@end

File: Sakura/Sakura/Extension/UIKit/UI/View/UIImageExtension.swift
//
//  UIImageExtension.swift
//  Sakura
//
//  Created by YaeSakura on 2016/11/24.
//  Copyright © 2016 Sakura. All rights reserved.
//

import Foundation

extension UIImage {
    
    public var centerStretchableImage : UIImage? {
        return self.stretchableImage(withLeftCapWidth: Int(self.size.width * 0.5), topCapHeight: Int(self.size.height * 0.5))
    }
    
} 
